src/VisualStudio/Core/Def/SymbolSearch/AbstractDelayStartedService.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.VisualStudio.LanguageServices.SymbolSearch
+{
+    internal abstract class AbstractDelayStartedService : ForegroundThreadAffinitizedObject
```

Comment:
"AbstractOptionalService"? Or at least a doc comment explaining what this is for?